src/resources/team/team.service.ts

```diff
@@ -134,33 +134,50 @@ export const createItem = async (data: Item): Promise<ItemDocument> => {
 
 	const newItem = await ItemModel.create(data)
 
-	return newItem
+	return newItem.populate('category').execPopulate()
```

Comment:
As we only need category name, it is better to populate only `name` field:
https://mongoosejs.com/docs/populate.html#field-selection